pkg/executor/infoschema_reader_test.go

```diff
@@ -1066,3 +1066,43 @@ func TestInfoschemaTablesSpecialOptimizationCovered(t *testing.T) {
 		require.Equal(t, testCase.expect, covered, testCase.sql)
 	}
 }
+
+func testInfoschemaTablesWhenTableIsNotPublic(t *testing.T, enableV2 bool) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	if enableV2 {
+		tk.MustExec("set @@global.tidb_schema_cache_size = 512 * 1024 * 1024")
+	} else {
+		tk.MustExec("set @@global.tidb_schema_cache_size = 0")
+	}
```

Comment:
This is unnecessary now?  We 70% chance run v2 and 30% chance v1 automatically (in testkit).
So there is no need to do it twice.

Reply:
It may be treated as a flaky test if it has the opportunity to success :facepalm:. BTW, by default, bazel will retry the failed tests (marked with `flaky = True`) three times, so it has more chances to success.

Therefore, I think it's better to actually run it twice.

Reply:
Not needed now.